Sources/PluginLibrary/NamingUtils.swift

```diff
@@ -366,7 +370,11 @@ public enum NamingUtils {
 
       let count = fromChars.distance(from: fromChars.startIndex, to: fromIndex)
       let idx = from.index(from.startIndex, offsetBy: count)
-      return from[idx..<from.endIndex]
+      #if swift(>=4.0)
+        return String(from[idx..<from.endIndex])
+      #else
+        return from[idx..<from.endIndex]
```

Comment:
Personally, I would omit the conditional here and only have the new code, which will work fine on Swift 3.